automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodImportSpec.scala

```diff
@@ -7,7 +7,7 @@ import org.broadinstitute.dsde.workbench.fixture._
 import org.broadinstitute.dsde.workbench.service.test.WebBrowserSpec
 import org.scalatest._
 
-class MethodImportSpec extends FreeSpec with ParallelTestExecution with Matchers with WebBrowserSpec with WorkspaceFixtures
+class MethodImportSpec extends FreeSpec /*with ParallelTestExecution*/ with Matchers with WebBrowserSpec with WorkspaceFixtures
```

Comment:
all Method's test suits cannot run in parallel due to test framework limitation.